.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,46 @@
+name: "Close Stale Issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 * * * *"
```

Comment:
Isn't every hour a little excessive? Seems to me that once a day would be plenty sufficient?

Reply:
I agree with this! Otherwise LGTM

Reply:
Sure, I copied the one from the CDK where I do think it's worth checking every hour because if someone responds I want labels to update pretty quickly so that they are not filtered out from the metrics email but we can make it once a day for jsii.

Reply:
I mean you could also time the daily run to happen an hour or so before the metrics are extracted? But that's up to you. I only mean to be conservative of free minutes of GitHub Actions executions :P